lib/gui/etcher.js

```diff
@@ -19,20 +19,10 @@
 const electron = require('electron')
 const _ = require('lodash')
 const path = require('path')
-const EXIT_CODES = require('../shared/exit-codes')
-let mainWindow = null
-
-electron.app.on('window-all-closed', electron.app.quit)
+const debug = require('debug')('gui:etcher')
 
-// Sending a `SIGINT` (e.g: Ctrl-C) to an Electron app that registers
-// a `beforeunload` window event handler results in a disconnected white
-// browser window in GNU/Linux and macOS.
-// The `before-quit` Electron event is triggered in `SIGINT`, so we can
-// make use of it to ensure the browser window is completely destroyed.
-// See https://github.com/electron/electron/issues/5273
-electron.app.on('before-quit', () => {
```

Comment:
But Ctrl-C keeps working, right?

Reply:
Yup, see a bit further down, at the `mainWindow.on('close')` handler

Reply:
Fair enough. There is the grey window thing described below, but other than that it works :+1: